Automattic/jetpack
# Social Icons: avoid notice with  new block-based Widget editor.

Fixes #20698

#### Changes proposed in this Pull Request:

* This adjustment should avoid PHP notices in some scenarios.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site with the Extra Sidebar widgets module active.
* Go to Appearance > Customize
* Add a new Social Icons (Jetpack) Widget
* Add some new services.
* Publish changes.
* Ensure your changes are visible on the frontend.
* Go back and edit your widget; you should be able to remove and edit values with no issues.
